src/configure/templateInputHelper/InputControl.ts

```diff
@@ -62,6 +62,24 @@ export class InputControl {
         this.visible = value;
     }
 
+    public updateInputControlType(inputMode: InputMode): void {
```

Comment:
I think we can move the logic to update and intialize the inputMode at one place in InputControlUtility and use it here as well as in InputControl.createControls()

Reply:
Updated